parallelTests.cmake

```diff
@@ -201,8 +201,8 @@ add_test_compare_parallel_simulation(CASENAME 3_a_mpi_multflt_mod2
                                      FILENAME 3_A_MPI_MULTFLT_SCHED_MODEL2
                                      SIMULATOR flow
                                      ABS_TOL ${abs_tol_parallel}
-                                     REL_TOL ${rel_tol_parallel}
-                               DIR model2
+                                     REL_TOL 1.0e-3
+                        			       DIR model2
```

Comment:
There is some extra white spaces or tabs here. Bu the original alignment was not great either. 

But since you are here, can you fix the alignment for this line? No worry if you do not do it, we can fix it easily.